packages/arm-core/src/utils/useObserveSize.ts

```diff
@@ -0,0 +1,53 @@
+import { RefObject, useLayoutEffect, useMemo, useState } from 'react';
```

Comment:
None of the other files have semicolons. This file does. Aren't there eslint rules in place for this?

Reply:
Yes, it looks like for some reason our linter does not like this file when running the lint task:

> TypeError: Cannot read property 'body' of null
> Occurred while linting /home/jon/Projects/amsterdam/amsterdam-react-maps/packages/arm-core/src/utils/useObserveSize.ts:29
>     at checkForConstructor (/home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/rules/no-useless-constructor.js:165:42)
>     at /home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/safe-emitter.js:45:58
>     at Array.forEach (<anonymous>)
>     at Object.emit (/home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
>     at NodeEventGenerator.applySelector (/home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/node-event-generator.js:254:26)
>     at NodeEventGenerator.applySelectors (/home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/node-event-generator.js:283:22)
>     at NodeEventGenerator.enterNode (/home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/node-event-generator.js:297:14)
>     at CodePathAnalyzer.enterNode (/home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:711:23)
>     at /home/jon/Projects/amsterdam/amsterdam-react-maps/node_modules/eslint/lib/linter/linter.js:952:32
>     at Array.forEach (<anonymous>)

Reply:
Should be fixed now.